UEBPrep/config/initializers/devise.rb

```diff
@@ -262,4 +262,6 @@
   # When using OmniAuth, Devise cannot automatically set OmniAuth path,
   # so you need to do it manually. For the users scope, it would be:
   # config.omniauth_path_prefix = '/my_engine/users/auth'
+
+  config.omniauth :facebook, "APP_ID", "APP_SECRET"
```

Comment:
@parkerBerger @stewarj Does this need to be replaced with the actual values for FB's APP_ID and APP_SECRET?